lib/pbio/platform/ev3/platform.ld

```diff
@@ -9,11 +9,17 @@ MEMORY
     SRAM_PRU1  (rw)  : ORIGIN = 0x80010000, LENGTH = 64K
     DDR_unused (rwx) : ORIGIN = 0xC0000000, LENGTH = 0x8000
     DDR        (rwx) : ORIGIN = 0xC0008000, LENGTH = (64M - 0x8000)
-    ARM_LRAM   (rwx) : ORIGIN = 0xFFFF0000, LENGTH = 8K
+    ARM_LRAM   (rwx) : ORIGIN = 0xFFFF0000, LENGTH = (8K - 16)
 }
 
 _minimal_stack_size = 4M;
 pbdrv_stack_end = ORIGIN(DDR) + LENGTH(DDR) - 4;
+/*
+We declare this in this style rather than creating a section.
+If we do create a section, the loader (U-boot) will always clear it.
```

Comment:
Even for a NOLOAD section?

Don't have a problem doing it this way, just curious.

Reply:
Yeah.

If loading via section headers (the default), sections flagged as `NOBITS` are cleared: https://github.com/pybricks/u-boot/blob/pybricks-ev3/lib/elf.c#L266-L268

If loading via program headers, the difference between filesz and memsz is cleared: https://github.com/pybricks/u-boot/blob/pybricks-ev3/lib/elf.c#L219-L221